dist/index.js

```diff
@@ -13616,7 +13617,9 @@ function run() {
                 core.setFailed("namespace is required and must be passed as an input from the workflow");
                 return;
             }
-            if (!(API_KEY && API_SECRET) && !GCP_CREDENTIALS_SERVICE_ACCOUNT) {
+            if (!ENABLE_GITHUB_ACTION_TOKEN &&
+                !(API_KEY && API_SECRET) &&
+                !GCP_CREDENTIALS_SERVICE_ACCOUNT) {
```

Comment:
at least one of them must be provided. The above checks if none of them is provided. But we need a check if at least one of them is provided.

Reply:
The check here is to fail and display an error if none of them is provided. If atleast one of them is provided, the run would proceed